sympy/calculus/singularities.py

```diff
@@ -44,6 +53,21 @@ def singularities(expression, symbol):
     >>> singularities(1/(y**3 + 1), y)
     {-1, 1/2 - sqrt(3)*I/2, 1/2 + sqrt(3)*I/2}
 
+    Returns
+    =======
+
+    Set
+        A set of values for `symbol` for which `expression` has a
+        singularity. An `EmptySet` is returned if `expression` has no
+        singularities for any given value of `Symbol`.
```

Comment:
Note that in RestructuredText if you want monospaced font you need to do: ` ``my text`` `